lib/enigma.rb

```diff
@@ -1,8 +1,24 @@
-class Enigma
-  attr_reader :shift_generator, :character_set
+require_relative 'cipher'
+
+class Enigma < Cipher
```

Comment:
enigma is a child of cipher, implemented so that future encrypters could theoretically exist.  for example a cesar cipher could also be a child of cipher